modules/doc/content/getting_started/installation/inl_hpc_install_moose.md

```diff
@@ -1,71 +1,172 @@
 # INL HPC Cluster
 
-The following instructions are for those operating on [!ac](INL) [!ac](HPC) machines.
+The following instructions are for those operating on [!ac](INL) [!ac](HPC) machines. This includes
+Bitterroot, Lemhi, and Sawtooth. It also includes the protected access hosts `rod` and `cone`.
 
-Requesting access (shell account) to [!ac](INL) [!ac](HPC) machines is handled by the
-[NCRC](https://inl.gov/ncrc/) group.
+Requesting access to [!ac](INL) [!ac](HPC) is handled by the [NCRC](https://inl.gov/ncrc/) group.
 
-For information on developing or running MOOSE-based NCRC applications (not the scope of this
-document), please head over to [NCRC Applications](help/inl/applications.md) instead, and choose the
+For information on running pre-built MOOSE-based NCRC applications (not the scope of this
+document), please see [NCRC Applications](help/inl/applications.md) instead, and choose the
 application applicable to you.
 
-## Environment id=environment
+## Containerization
 
-!style! halign=left
-While operating on one of our [!ac](INL) [!ac](HPC) clusters, you need only load a couple of
-modules to obtain a proper developer environment:
-!style-end!
+The development environment is deployed using containerization, which is a deployment
```

Comment:
```suggestion
!style halign=left
The development environment is deployed using containerization, which is a deployment
```
Like we talked about, and everywhere else this should be added. Or, of course, if you have a global CSS fix, that would be fantastic too.